File: WarehouseManagementWeb.Application/Dto/Output/Client/ClientOutput.cs
using WarehouseManagementWeb.Domain.Enums;

namespace WarehouseManagementWeb.Application.Dto.Output.Client
{
    /// <summary>
    /// Класс выходной модели клиентов.
    /// </summary>

    public class ClientOutput
    {
        /// <summary>
        /// Id клиента.
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        /// Наименование клиента.
        /// </summary>
        public string? Name { get; set; }

        /// <summary>
        /// Адрес клиента.
        /// </summary>
        public string? Address { get; set; }

        /// <summary>
        /// Статус клиента в значении перечисления.
        /// </summary>
        public DirectoryStatusEnum ClientStatusEnum { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/directory/client/models/output/client-output.ts
import { DirectoryStatusEnum } from '../../../enums/directory-status-enum';

/**
 * Класс выходной модели клиентов.
 */
export class ClientOutput {
  /**
   * Id клиента.
   */
  id: number = 0;

  /**
   * Наименование клиента.
   */
  name: string = '';

  /**
   * Адрес клиента.
   */
  address: string = '';

  /**
   * Статус клиента.
   */
  clientStatusEnum: DirectoryStatusEnum = 0;
}
